docs/faq/network/network-configuration.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_label: 'Network Configuration'
+sidebar_position: 2
+---
+
+# Network Configuration
+
+### Will lower latency positively influence my rewards?
+
+There are no extra rewards for being very quick. When the appointed time for a validator comes to propose a block, the validator has 12 seconds to vote. The validator will only miss the slot if the node is not synced up or offline during this time.
+
+### Which ports must be open on a node machine?
+
+The following ports will need to be opened to run the blockchain node correctly:
+
+| CLIENT     | DESCRIPTION                               | PORT  | TCP | UDP |
+| ---------- | ----------------------------------------- | ----- | --- | --- |
+| GETH       | Execution Chain Data Channel              | 30303 | X   |     |
+| GETH       | Execution Chain Discovery                 | 30303 |     | X   |
+| ERIGON     | Execution Chain Data Channel              | 30303 | X   |     |
+| ERIGON     | Execution Chain Discovery                 | 30303 |     | X   |
+| LIGHTHOUSE | Beacon Communication and Data             | 9000  | X   | X   |
+| PRYSM      | Beacon Gossip, Requests, and Responses    | 13000 | X   |     |
+| PRYSM      | Beacon Discovery, Requests, Data Exchange | 12000 |     | X   |
```

Comment:
I will remove the client + description. Just answer the question:
- which ports must be open?

30303 TCP + UDP
12000 UDP
13000 TCP
(Lighthouse only) 9000 TCP + UDP

Reply:
if we remove the description and clients, there will be more questions about why they need to be open and which clients they are used for. we need to create these answers so that there are no further questions, freeing our resources.

IMO we should keep them